### PR TITLE
Blueprinting 3

### PR DESCRIPTION
**Pull request by martinavagyan**
Fixed local run and Heroku server 
Also enhancements from issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ An adaptive online language practice platform
 =====
 [![Build Status](https://travis-ci.org/martinavagyan/zeeguu-exercises.svg?branch=master)](https://travis-ci.org/martinavagyan/zeeguu-exercises)
 [![Coverage Status](https://coveralls.io/repos/github/martinavagyan/zeeguu-exercises/badge.svg?branch=master)](https://coveralls.io/github/martinavagyan/zeeguu-exercises?branch=master)
+[![Dependency Status](https://gemnasium.com/badges/github.com/martinavagyan/zeeguu-exercises.svg)](https://gemnasium.com/github.com/martinavagyan/zeeguu-exercises)
```
Comment (mircealungu) [inline README.md]: whoa. dependency status :)
you're leading in terms of badges in your project at the moment :)

File: src/zeeguu_exercises/static/js/app/exercises/exercise.js [hunk 2]
```diff
@@ -140,7 +151,7 @@ Exercise.prototype = {
 		//Calculate time taken for single exercise
 		var exTime = Util.calcTimeInMilliseconds(this.startTime);
 		//Request back to the server with the outcome
-		console.log(Settings.ZEEGUU_API + Settings.ZEEGUU_EX_OUTCOME_ENDPOINT + exOutcome +  _this.resultSubmitSource + "/" + exTime + "/" + id + "?session="+this.session);
+		//console.log(Settings.ZEEGUU_API + Settings.ZEEGUU_EX_OUTCOME_ENDPOINT + exOutcome +  _this.resultSubmitSource + "/" + exTime + "/" + id + "?session="+this.session);
```
Comment (mircealungu) [inline src/zeeguu_exercises/static/js/app/exercises/exercise.js]: do you need the comment here?

File: src/zeeguu_exercises/static/styles/custom.css [hunk 3]
```diff
@@ -240,10 +241,9 @@ body
 }
 
 .clickable-symbol {
-	margin:auto;
-    width: 45px;
-    height: 45px;	
-	margin-top: 15px;
+	width: 45px;
+    height: 45px;
```
Comment (mircealungu) [inline src/zeeguu_exercises/static/styles/custom.css]: strange align.

File: README.md (hunk 1 above)
Comment (martinavagyan) [inline README.md]: they decorate nicely :)

File: src/zeeguu_exercises/static/js/app/exercises/exercise.js (hunk 2 above)
Comment (martinavagyan) [inline src/zeeguu_exercises/static/js/app/exercises/exercise.js]: not needed, just for me when debugging

File: README.md (hunk 1 above)
Comment (mircealungu) [inline README.md]: yes! even if I'm not sure about the dependencies: out of date one... :)

File: src/zeeguu_exercises/static/styles/custom.css (hunk 3 above)
Comment (martinavagyan) [inline src/zeeguu_exercises/static/styles/custom.css]: yeah, indeed, on the IDE it does not show this way

**Comment by mircealungu** [comment]
okok!

File: README.md (hunk 1 above)
Comment (mircealungu) [inline README.md]: however, I didn't know about Gemnasium. 
Looks quite cool actually! 
Nice find!!

Comment (martinavagyan) [inline README.md]: 👍

**Pull request merged**